arch/arm64/boot/dts/qcom/msm8953-xiaomi-tissot.dts

```diff
@@ -56,34 +87,119 @@
 			pmsg-size = <0x8000>;
 		};
 	};
-};
 
-&max98927_codec {
-	status = "okay";
+	i2c-sensors {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 14 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 15 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		i2c-gpio,delay-us = <2>; /* ~100 kHz */
+		#address-cells = <1>;
+		#size-cells = <0>;
 
-	reset-gpios = <&tlmm 86 GPIO_ACTIVE_LOW>;
+		ak09918: magnetometer@c {
+			compatible = "asahi-kasei,ak09918";
+			reg = <0x0c>;
+
+			vdd-supply = <&pm8953_l10>;
+			vid-supply = <&pm8953_l6>;
+		};
+
+		/*
+		 * Driver still missing
+		 * Just for general information
+		 */
+		ltr579: light@53 {
```

Comment:
tissot seems to report the same sensor of vince, also on hwdeviceinfo. But I'm successfully using CONFIG_LTRF216A and this snippet in dts
```
        light-sensor@53 {
            compatible = "liteon,ltrf216a";
            reg = <0x53>;
            vdd-supply = <&pm8953_l10>;
        };
```
You should be able to obtain values with 
`cat /sys/bus/iio/devices/iio:device0/in_illuminance_raw`